sed/tests/test.bats

```diff
@@ -0,0 +1,22 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+pkg_ident=$(grep pkg_ident $BATS_TEST_DIRNAME/../results/last_build.env | cut -d \= -f2)
```

Comment:
We should be able to source `last_build.env` here and have $pkg_ident set.  Is there a reason we don't want the remainder of the variables in that file in our test environment?

Reply:
I'll change this. I think this was because I previously wrote the tests in InSpec and InSpec isolates its environment.